### ISSUE TITLE
Change in install/config. workflow...

### ISSUE THREAD

**Issue by dkossack** [symless/synergy#5557]
Server & Clients OS-X 10.10.5
I just installed synergy-v1.8.3-rc1-e36a7ae-MacOSX1010-x86_64 for beta testing:

1. When I do a new install my clients end up getting disconnected from the server and I end up accessing the clients directly to reconnect, usually simply with a click of the 'Apply' button... This has been with all of my update/upgrades.

2. With update 1.8.3-rc1 I found that the clients had to be started twice:
a. install;
b. configure;
c. the GUI tells me to close the Wizard and Synergy will continue in background... 
d. I close, Synergy works but I don't get the Synergy icon in the top menu bar (so no access to the preferences or );
e. restarting Synergy from Application/Synergy.app brings up the 'final' window on the Wizard with a 'Done' button;
f. a click on 'Done' brings up the icon and access to start/stop, show/hide and quit pulldowns.

3. Clicking 'Apply' on the Server GUI disconnects clients:

Server: click 'Apply' 
[2016-08-22T16:18:02] INFO: starting server
...
([2016-08-22T16:27:13] NOTE: started server, waiting for clients)
2016-08-22 16:31:01.326 synergys[21115:1721250] starting cocoa loop

Client (responds):
[2016-08-22T16:35:02] ERROR: ssl error occurred (system call failure)
[2016-08-22T16:35:02] ERROR: eof violates ssl protocol
[2016-08-22T16:35:03] NOTE: disconnected from server

on Client: click 'Apply'
[2016-08-22T16:35:09] INFO: stopping synergy desktop process
[2016-08-22T16:35:09] INFO: process exited normally
[2016-08-22T16:35:09] INFO: starting client
[2016-08-22T16:35:09] INFO: config file: /var/folders/8v/ytz36jf50096y980cxjt4yq80000gn/T/Synergy.O10326
[2016-08-22T16:35:09] INFO: log level: INFO
[2016-08-22T16:35:09] INFO: log file: /Users/david/Desktop/synergy.log
[2016-08-22T16:35:09] INFO: drag and drop enabled
[2016-08-22T16:35:09] NOTE: started client
[2016-08-22T16:35:09] NOTE: connecting to '10.0.0.45': 10.0.0.45:24800
[2016-08-22T16:35:09] INFO: OpenSSL 1.0.2 22 Jan 2015
2016-08-22 16:35:09.494 synergyc[10396:827555] starting cocoa loop
[2016-08-22T16:35:09] NOTE: server fingerprint: 03:B5:38:9E:92:08:78:1D:64:4D:62:9D:F7:41:59:7A:98:9E:C0:52
[2016-08-22T16:35:09] INFO: connected to secure socket
[2016-08-22T16:35:09] INFO: server ssl certificate info: /CN=Synergy
[2016-08-22T16:35:09] INFO: AES256-GCM-SHA384       TLSv1.2 Kx=RSA      Au=RSA  Enc=AESGCM(256) Mac=AEAD
[2016-08-22T16:35:09] WARNING: cursor may not be visible
[2016-08-22T16:35:09] NOTE: connected to server

Server Debug Log file:
[synergys1.log.txt](https://github.com/symless/synergy/files/431405/synergys1.log.txt)
Client Debug Log file:
[synergyc1.log.txt](https://github.com/symless/synergy/files/431407/synergyc1.log.txt)

**Comment by XinyuHou** [comment]
@dkossack 
Thank you for beta testing!

> the GUI tells me to close the Wizard and Synergy will continue in background...
> estarting Synergy from Application/Synergy.app brings up the 'final' window on the Wizard with a 'Done' button

Do you mean after you finish wizard, it doesn't allow you to click finish?

**Comment by dkossack** [comment]
On the Client side, after using Jerry’s tools to clean previous install configurations, the latest beta install (1.8.2 > 1.8.3) allowed me to close the Wizard but did not present the Synergy icon in the toolbar at the top of the Desktop… when I quit Synergy and reopened from Synergy.app the Wizard popped up at the last page with the ‘Done’ button. I clicked ‘done’ and the Wizard closes and the icon appears… as it should have.

I was not able to replicate this event when I tried it just now (./clean*.sh; install 1.8.3.dmg over 1.8.3) but it occurred on both of my clients on the initial install.

I hope that helps…

David
